graph/src/data/query/error.rs

```diff
@@ -32,7 +32,7 @@ pub enum QueryExecutionError {
     OrderByNotSupportedError(String, String),
     OrderByNotSupportedForType(String),
     FilterNotSupportedError(String, String),
-    UnknownField(Pos, String, String),
+    UnknownField(String, String),
```

Comment:
I removed it because it wasn't used in the error mesage, but I'll put it back.

Reply:
It's true it's not used in `Display` but it could be used in `Deserialize` further below. Can we add this variant there? See https://github.com/graphprotocol/graph-node/blob/90ce7e355bf6d42ca92a9be5e9381e6e8e1f917d/graph/src/data/query/error.rs#L326